src/import/chips/p9/procedures/xml/attribute_info/p9_pcie_attributes.xml

```diff
@@ -120,6 +120,7 @@
     <valueType>uint8</valueType>
     <platInit/>
     <persistRuntime/>
+    <writeable></writeable>
```

Comment:
Instead of changing the value here, add an entry to src/usr/targeting/common/xmltohb/hb_customized_attrs.xml, that maintains the read-only property of the attribute in the FAPI layer while still allowing Hostboot to modify it.

Reply:
Tested it can work, will change it.